src/main/java/com/vmware/vipclient/i18n/messages/api/opt/local/LocalMessagesOpt.java

```diff
@@ -67,9 +67,11 @@ public void getComponentMessages(MessageCacheItem cacheItem) {
 	private InputStream getInputStream() {
 		String locale = LocaleUtility.fmtToMappedLocale(dto.getLocale()).toLanguageTag();
 		while (true) {
-			String filePath = FormatUtils.format(OFFLINE_RESOURCE_PATH, dto.getComponent(), locale);
-			Path path = Paths.get(VIPCfg.getInstance().getOfflineResourcesBaseUrl(), filePath);
-			InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream(path.toString());
+			String offlineResourcePath = VIPCfg.getInstance().getOfflineResourcesBaseUrl();
+			if(!offlineResourcePath.endsWith("/"))
```

Comment:
Can "/" be suitable for all systems? Should it be FileSeparator?

Reply:
No, that is the bug. If we use FileSeparator or Paths.get, '\' (backslash) will be used in Windows platform. However, the forward slash in jar is not platform dependent. Even if the application is run on a Windows platform, the jar files will still have forward slash. The use of Paths.get (same as FileSeparator) was causing the issue.